src/avatar/network.js

```diff
@@ -0,0 +1,186 @@
+createNameSpace("realityEditor.avatar.network");
+
+/**
+ * @fileOverview realityEditor.avatar.network
+ * Contains a variety of helper functions for avatar/index.js to create and discover avatar objects,
+ * realtime broadcast my avatar's state, and subscribe to the state of other avatars
+ */
+
+(function(exports) {
+    const DATA_SEND_FPS_LIMIT = 60;
```

Comment:
This should be lower to play nice with the cloud proxy. Usually I've been going with 10 fps but this might be able to go a bit higher like 30 or so. The only issue with having it be this high is that there's no backpressure on the socket so the server can't say "hey slow this down while I recover"

Reply:
I'll try 30 FPS, if you have issues with it feel free to lower it back to 10